jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_sensors/openni.launch

```diff
@@ -0,0 +1,118 @@
+<!-- Entry point for using OpenNI devices -->
```

Comment:
if this is copied from  `(find openni_launch)/launch/openni.launch` please add commen in the top of this file and add what did you changed with reason.

Reply:
I updated and added the comments